### PR TITLE
Starting to check commits

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

File: .travis/bootstrap_catdog.sh [hunk 1]
```diff
@@ -20,4 +22,4 @@ cp ../plugin_template/.travis/post_before_script.sh .travis/
 # generate a git commit
 git init
 git add .
-git commit -m $'Initial commit\n\n[noissue]'
+git commit -m "${COMMIT_MSG}"
```
Comment (daviddavis) [inline .travis/bootstrap_catdog.sh:25]: It feels like a hack having the plugin bootstrap test check plugin_template's commit messages. For one thing, I am guessing it doesn't handle multiple commits?

What if we convert templates/travis/.travis/validate_commit_message.py.j2 to templates/travis/.travis/validate_commit_message.py (the only thing we use jinja for in this file is to add a header which I don't think we need) and then write a small github workflow or travis job to run it?

Comment (fao89) [reply .travis/bootstrap_catdog.sh:28]: it is for doing the `Required PR` thing, and it only checks the last commit

Comment (daviddavis) [reply .travis/bootstrap_catdog.sh:28]: Don't we want to validate all commits?

Comment (daviddavis) [reply .travis/bootstrap_catdog.sh:28]: It also still feels weird to me that we're using the bootstrap test code to validate commit messages.

File: README.md [hunk 2]
```diff
@@ -487,3 +487,15 @@ optional arguments:
...
 - [ ] Errors are handled according to Pulp conventions
 - [ ] Docs for plugin are available (any location and format preferred and provided by plugin writer)
+
+
+# Contributing
+
+### Pull Request Checklist
+
+1. Unless a change is small or doesn't affect users, create an issue on
+https://pulp.plan.io/projects/pulp . Set the Category to "Installer".
```
Comment (daviddavis) [inline README.md:497]: s/affect uesrs/affect plugin writers/
s/Set the Category to "Installer"//Add the tag "Plugin Template"./

Also remove the unnecessary space before the period.

**Pull request merged**